contrib/hadoop/src/main/java/com/google/cloud/dataflow/contrib/hadoop/HadoopFileSource.java

```diff
@@ -432,6 +432,12 @@ private Double getProgress() {
     }
 
     @Override
+    public final long getParallelismRemaining() {
+      // Not yet supported. See #splitAtFraction.
```

Comment:
Better to say "This source currently doesn't support dynamic work rebalancing, so remaining parallelism is always 1".